beetmoverscript/test/test_work_dir/task.json

```diff
@@ -53,6 +53,25 @@
         }
       ]
     },
+    "extra": {
```

Comment:
AFAIK, this is usefless as it's not processed in beetmover. But most likely you've done a copy-paste from a real task so this certainly doesn't hurt 👍